src/main/java/uk/gov/hmcts/reform/pip/account/management/controllers/AccountController.java

```diff
@@ -130,4 +130,13 @@ public ResponseEntity<Map<CreationEnum, List<?>>> createMediaAccountsBulk(
         @RequestHeader("x-issuer-id") String issuerId, @RequestPart MultipartFile mediaList) {
         return ResponseEntity.ok(accountService.uploadMediaFromCsv(mediaList, issuerId));
     }
+
+    @ApiResponses({
+        @ApiResponse(code = 202, message = "Account Management - MI Data request accepted.")
+    })
+    @ApiOperation("Returns a list of (anonymized) account data for MI reporting.")
+    @GetMapping("/mi-data")
+    public ResponseEntity<String> getSubscriptionDataForMiReportingLocal() {
```

Comment:
There method name should be "getAccountData..." not "subscription"

Reply:
Eek, good shout!